__tests__/api.test.js

```diff
@@ -44,3 +45,18 @@ describe("/api/categories", () => {
       });
   });
 });
+
+describe("Returns with the correct JSON endpoint", () => {
+  test("/api/categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((response) => {
+        const input = Object.keys(
+          endpoint["GET /api/categories"].exampleResponse
+        );
+        const expectedOutput = Object.keys(response.body);
+        expect(input).toMatchObject(expectedOutput);
```

Comment:
This is much better!
Points for improvement: 
You could iterate through the returned array and be more specific about what form the JSON object should take.

Maybe have a look at [.toMatchObject( )](https://jestjs.io/docs/expect#tomatchobjectobject) and [expect.objectContaining( )](https://jestjs.io/docs/expect#expectobjectcontainingobject)

Reply:
Thank you I'll merge this one and have a refactor for the next one.